mlops_utilities/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+Python library provides implementation for a few high level operations
+most commonly occuring in any MLOps workflows built in AWS
+"""
+
+from __future__ import annotations
+
+# Set default logging handler to avoid "No handler found" warnings.
+import logging
+import typing
+import warnings
+from logging import NullHandler
+
+logging.getLogger(__name__).addHandler(NullHandler())
```

Comment:
WTF is this?

Reply:
See my comments
- https://github.com/provectus/mlops-utilities/pull/17#issuecomment-1370917059
- https://github.com/provectus/mlops-utilities/pull/17#discussion_r1062252108

Reply:
looks good